content/en/docs/concepts/storage/volumes.md

```diff
@@ -1115,6 +1115,25 @@ persistent volume:
   secret is required. If the secret object contains more than one secret, all
   secrets are passed.
 
+#### CSI raw block volume support
+
+{{< feature-state for_k8s_version="v1.11" state="alpha" >}}
+
+Starting with version 1.11, CSI introduced support for raw block volumes.  This work
+relies on the raw block volume feature that was introduced in previous versions of 
+Kubernetes.  This feature will make it possible for vendors with external CSI drivers to 
+implement raw block volumes support in Kubernetes workload.
+
+CSI block volume support is feature-gated and turned off by default.  To run CSI with
+block volume support enabled, add the following feature gate flag:
+
+```
+--feature-gates=CSIBlockVolume=true
+```
+
+Learn more about raw block volume support and see how to setup your PV/PVC 
+[here](/docs/concepts/storage/persistent-volumes/#raw-block-volume-support).
```

Comment:
In the referenced page, there is a note:

> Only statically provisioned volumes are supported for alpha release.

Is this still true for block volumes? Is it true for CSI volumes which encourages dynamic provisioning?

Reply:
@tengqm I will check on this as well.